Control coverage/Feature/ApplicationGateway.md

```diff
@@ -6,7 +6,7 @@
 
 - [Azure_ApplicationGateway_NetSec_Enable_WAF_Configuration](#azure_applicationgateway_netsec_enable_waf_configuration)
 - [Azure_ApplicationGateway_NetSec_Enable_DDoS_Protection](#azure_applicationGateway_netsec_enable_ddos_protection)
-
+- [Azure_ApplicationGateway_DP_Enable_EndToEnd_Encryption_Trial](#Azure_ApplicationGateway_DP_Enable_EndToEnd_Encryption_Trial)
```

Comment:
I thought we didn't put documentation for trial controls? Is this correct?

Reply:
Removed trial